tools/run_local_tests.sh

```diff
@@ -33,12 +34,65 @@ while true ; do
   sleep 1
 done
 
-echo "+ Creating database ${KBS_DB}"
+echo "+ Creating mysql database ${KBS_DB}"
 mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} -e "CREATE DATABASE ${KBS_DB};"
 mysql -u${KBS_DB_USER} -p${KBS_DB_PW} -h ${KBS_DB_HOST} ${KBS_DB} < "${SIMPLE_KBS_DIR}/db.sql"
 
-echo "+ Running: cargo test"
+echo "+ Running: cargo test for mysql using mariadb:latest"
 cargo test
 
-echo "+ Stopping DB server container..."
+echo "+ Stopping mariadb:latest DB server container..."
 docker stop kbs-db
+
+
+export KBS_DB_USER=postgres
+export KBS_DB_PW=root
+export KBS_DB=simple_kbs
+export KBS_DB_TYPE=postgresql
+
+export POSTGRES_PASSWORD=$KBS_DB_PW \
+export POSTGRES_USER=$KBS_DB_USER \
+export PGPASSWORD=$KBS_DB_PW \
+
+echo "+ Starting postgres:latest DB server container..."
+docker run --detach --rm \
+           -p 5432:5432 \
+           --name kbs-db-postgres \
+           --env POSTGRES_PASSWORD=$KBS_DB_PW \
+           --env POSTGRES_USER=$KBS_DB_USER \
+           --env PGPASSWORD=$KBS_DB_PW \
+           postgres:latest
+
+sleep 5
+echo -n "+ Trying to connect to postgres DB server..."
+while true ; do
+  echo -n '.'
+  if psql -U ${KBS_DB_USER} -h ${KBS_DB_HOST} -c "SELECT 1;" > /dev/null 2>&1 ; then
+    echo " Success!"
+    break
+  fi
+  sleep 1
+done
+
+echo "+ Creating postgres database ${KBS_DB}"
+psql -U ${KBS_DB_USER} -h ${KBS_DB_HOST} -c "CREATE DATABASE ${KBS_DB};"
+psql -U ${KBS_DB_USER} -h ${KBS_DB_HOST} ${KBS_DB} < "${SIMPLE_KBS_DIR}/db.postgres.sql"
+
+echo "+ Running: cargo test for postgres using postgres:latest"
+cargo test
+
+echo "+ Stopping DB server container..."
+docker stop kbs-db-postgres
+
+echo "+ Updating exports for sqlite testing..."
+
+
+export KBS_DB_TYPE=sqlite
+export KBS_DB_HOST=localhost
+export KBS_DB_USER=${USER}
+export KBS_DB_PW=${USER}
```

Comment:
I suggest setting to empty strings to indicate they are not used:

```suggestion
export KBS_DB_HOST=
export KBS_DB_USER=
export KBS_DB_PW=
```

Reply:
This won't work with the current version of get_dbpool, but I will fix them in a future PR.  The reason is that we check if these are set and fail if they don't.